android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/BackButton.kt

```diff
@@ -0,0 +1,44 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.Transparent
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+
+private const val Rotate180 = 180f
+
+@Composable
+fun BackButton(
+    modifier: Modifier,
+    onBackClick: () -> Unit
```

Comment:
optional: `onClick` is enough 😄

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/65/commits/143521ec9a3fee7dc664e8269e61f340995767f5